### ISSUE TITLE
Cookie Warning to ensure EU regulatory compliance

### ISSUE THREAD

**Issue by robertgdev** [zikula/core#728]
As of last year, a new EU law came into effect which basically says that you have to tell your users that you will be using cookies which are saved on their local computers. Failing to do so could leave you liable for a lawsuit (or an "Abmahnung" in Germany) and force you to pay a fine. 

The way this can play out can be along two lines
1) Have the user choose to accept cookies; if he doesn't, run without cookies (can we even do this)? 
2) Simply tell the user that we're using cookies and tell him that he should not use the site if he does not agree to our use of cookies.
#1 is nicer, but I'm not sure if that's easily possible.

Bottom Line: Zikula should offer a nice-looking method of alerting users to the use of cookies out of the box in order to ensure regulatory compliance in the EU.
